main.py

```diff
@@ -4,6 +4,8 @@
 from fastapi import FastAPI
 from fastapi.responses import JSONResponse
 
+from utils import generate_master_mind_result
```

Comment:
since you created a new file, better put everything into one module and use relative imports